content/applications/general/voip/axivox/vm_audio_messages.rst

```diff
@@ -0,0 +1,262 @@
+Under :guilabel:`Extension to use for message management` field, click the drop-down menu, and
+select the extension where Axivox should call to record the message.
+
+Then, click :guilabel:`OK` to begin the call.
+
+.. note::
+   The user **must** be active in the production database with :abbr:`VoIP (Voice over Internet
+   Protocol)` configured. To configure :abbr:`VoIP (Voice over Internet Protocol)` for a user, see
+   this documentation: :doc:`axivox_config`.
+
+Upon connecting to the Axivox audio recorder management line, a recorded French-speaking operator
+provides the following options:
+
+#. Press `1` to record a message.
+#. Press `2` to listen to the current message.
+
+Press either `1` or `2`, depending on whether or not there is already a message present in the
+system for this particular audio message that requires a review, before recording a new one.
+
+Record the new audio message after pressing `1`, then press `#` to end the recording.
+
+The French-speaking operator returns to the line presenting the first set of questions again:
+
+#. Press `1` to record a message.
+#. Press `2` to listen to the current message.
+
+Press `#` to end the call.
+
+Write audio message
+-------------------
+
+To type the message, and select a computerized speaker to say the text, navigate to the
+:menuselection:`Audio messages` in the menu on the left.
+
+From the :guilabel:`Audio messages` page, select the blue button labeled, :guilabel:`Text message`,
+next to the corresponding audio message :guilabel:`Name` that the message should be attached to.
+
+Doing so reveals :guilabel:`Convert text to message` pop-up window.
+
+From the :guilabel:`Convert to text message` pop-up window, click the drop-down menu next to the
+field labeled, :guilabel:`Voice`, and select an option for the :guilabel:`Text` to be read in.
+
+After the :guilabel:`Voice` selection has been made, and the message has been written in the
+:guilabel:`Text` field, click :guilabel:`Generate` to process the audio file.
+
+The text is read in the same language it is written in the :guilabel:`Text` field. Should the
+language differ in the :guilabel:`Voice`, then an accent is used by the computerized speaker.
+
+Finally, when these steps are complete, click :guilabel:`Save` to save the audio message.
+
+To implement the changes, click :guilabel:`Apply changes` in the upper-right corner of the screen.
+
+.. image:: vm_audio_messages/sample-message.png
+   :align: center
+   :alt:  Convert text to message window with voice, text, generate button and save highlighted.
+
+.. tip::
+   To set a greeting or audio message in a dial plan element double click on the element. This could
+   be a :guilabel:`Play a file` or even a :guilabel:`Menu` element in which the caller should
+   encounter an urgent message or a dial-by-number directory. For more information on  dial plans
+   see this documentation: :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.
```

Comment:
```suggestion
   To set a greeting or audio message in a dial plan element double-click on the element. This could
   be a :guilabel:`Play a file` element, or a :guilabel:`Menu` element, in which the caller should
   encounter an urgent message, or a dial-by-number directory.
   
   For more information on dial plans see this documentation: :doc:`dial_plan_basics` or
   :doc:`dial_plan_advanced`.
```